docs/storage-drivers/gcs.md

```diff
@@ -44,6 +44,17 @@ An implementation of the `storagedriver.StorageDriver` interface which uses Goog
       A private service account key file in JSON format. Instead of a key file <a href="https://developers.google.com/identity/protocols/application-default-credentials">Google Application Default Credentials</a> can be used.
     </td>
   </tr>
+  <tr>
+    <td>
+      <code>credentials</code>
+    </td>
+    <td>
+      no
+    </td>
+    <td>
+      The content of the JSON format key file. The keyfile argument has higher priority than the credentials argument if both are specified.
```

Comment:
Yes.  It's a misconfiguration to supply both since one will be completely overridden.  It's not intuitive that the key file overrides the contents of the configuration file.